pkg/operation/botanist/monitoring.go

```diff
@@ -47,8 +46,7 @@ import (
 )
 
 const (
-	secretNameIngressOperators = v1beta1constants.SecretNameObservabilityIngress
-	secretNameIngressUsers     = v1beta1constants.SecretNameObservabilityIngressUsers
+	secretNameIngressUsers = v1beta1constants.SecretNameObservabilityIngressUsers
```

Comment:
```suggestion
	secretNameIngress = v1beta1constants.SecretNameObservabilityIngressUsers
```

Reply:
I adjusted the variable names in the commit https://github.com/gardener/gardener/pull/7007/commits/5e68b43d523fee502f8487b61c30175a3c4377eb
"Remove the `Users` infix/suffix from two variable names"